shared/src/main/scala/StringHelpers.scala

```diff
@@ -54,4 +124,52 @@ object StringHelpers:
       if index == -1 then c
       else mapping((index + 1) % mapping.length)
     }.mkString
+
+  // https://github.com/DennisMitchell/jellylanguage/blob/70c9fd93ab009c05dc396f8cc091f72b212fb188/jelly/interpreter.py#L1055
+  def sss(compressed: String)(using ctx: Context): String =
+    val decompressed = StringBuilder()
+    var integer =
+      NumberHelpers
+        .fromBase(
+          VList.from(compressed.map(CODEPAGE.indexOf(_): VNum)),
```

Comment:
No clue how scala works, but is the `+1` not needed here? ([...](https://github.com/DennisMitchell/jellylanguage/blob/70c9fd93ab009c05dc396f8cc091f72b212fb188/jelly/interpreter.py#L1058))

Reply:
I missed that part of the original. It needs to be added